qos/evm/check_empty.go

```diff
@@ -0,0 +1,41 @@
+package evm
+
+import (
+	"errors"
+)
+
+var _ check = &endpointCheckEmptyResponse{}
+
+// TODO_MVP(@commoddity): should we provide for a mechanism to un-sanction an endpoint that has returned an empty response?
```

Comment:
I think we should remove this.

This is why we have session & permanent sanctions. 

In addition, restarts cancel permanent sessions.

I feel like we're getting into bikeshedding territory with this comment for now. If/when the time to revisit this comes up, I'm sure it'll float up on its own.